### DISCUSSION TITLE
MacAndMayo's Skin Packs

### DISCUSSION THREAD

**Discussion by CursedBM81** [RedRain0o0/legacy-skins#4] [(A) General Skin Packs]
These are the packs I ported from LCE to LS.
[1st Birthday.zip](https://github.com/user-attachments/files/16827529/1st.Birthday.zip)
[Skin Pack 1.zip](https://github.com/user-attachments/files/16827543/Skin.Pack.1.zip)
[Skin Pack 3.zip](https://github.com/user-attachments/files/16827546/Skin.Pack.3.zip)
[PlayStation Pack 1.zip](https://github.com/user-attachments/files/16827547/PlayStation.Pack.1.zip)
[PlayStation Pack 2.zip](https://github.com/user-attachments/files/16827550/PlayStation.Pack.2.zip)
[PlayStation Pack 3.zip](https://github.com/user-attachments/files/16827553/PlayStation.Pack.3.zip)

There's also a custom skin pack I made that has a few classic Minecraft YouTubers, excluding anybody with a SunnyV2 video on them.
[Classic MCYT Skin Pack.zip](https://github.com/user-attachments/files/16827555/Classic.MCYT.Skin.Pack.zip)

**Comment by jakeyhanna** [comment]
How did you make the model appear in the menu??

When i try to make a pack with my cpmmodel the skin doesn't render but it works fine
